research-software-directory/RSD-as-a-service
# 195 project add

# Add project

Fixes #195 

Changes proposed in this pull request:
* User is able to add new projet to RSD

How to test:
* `docker-compose down ---volumes`: remove old data
* `docker-compose build`: build all
* `docker-compose up`: start solution
*  Login 
* You should have option in the top right corner "New project"
![image](https://user-images.githubusercontent.com/9204081/163578553-81606324-acfd-4c8a-8eea-b44db48f22d0.png)
* Add project, test same slug twice (will warn you that slug is taken)
![image](https://user-images.githubusercontent.com/9204081/163578657-63a417fe-53ba-4b9e-a37f-55075629959b.png)
* After saving you are forwarded to edit page. This page is under construction and will have only slug value in the title

PR Checklist:
